contrib/solr/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# ARG used in FROM must come first
+ARG SOLR_VERSION=7.5.0
+
+FROM busybox as buildenv
+
+# ARG for build must come after the FROM statement
+ARG XWIKI_VERSION=10.9
+ARG XWIKI_URL_PREFIX="https://maven.xwiki.org/releases/org/xwiki/platform/xwiki-platform-search-solr-server-data/${XWIKI_VERSION}"
+
+WORKDIR /var/xwiki-solr-temp/
+
+COPY ./solr-init.sh .
```

Comment:
What is this doing and where is solr-init.sh coming from?

Reply:
`solr-init.sh` was already there and has not been changed.
See: https://github.com/xwiki-contrib/docker-xwiki/blob/master/contrib/solr/solr-init.sh
Look into the file, it has comments. It looks for the *.jar and unzips it to `/opt/solr/server/solr/`